runtime/runtime.go

```diff
@@ -0,0 +1,13 @@
+package runtime
+
+// Enumeration of network upgrades where actor behaviour can change (without necessarily
+// vendoring and versioning the whole actor codebase).
+type NetworkVersion uint // FIXME move to types
+
+const (
+	NetworkVersion0 = NetworkVersion(iota) // specs-actors v0.9.3
+	NetworkVersion1                        // specs-actors v0.9.7
+	NetworkVersion2                        // specs-actors v2.0.?
```

Comment:
`NetworkVersion2` will now be another `0.9.x`.

Please add `NetworkVersion3` too to save us a PR later.